app/src/main/res/values-ja/strings.xml

```diff
@@ -360,6 +360,7 @@
   <string name="device_offline_dialog_negative">消去</string>
   <string name="device_offline_dialog_positive">設定</string>
   <string name="saving">保存中…</string>
+  <string name="scan_first_product">Scan First Product</string>
```

Comment:
The translated string is same  if you check it in Japanese translations or Any other translations.  should we change this string to a `translatable:false`  Rather than the whole new 151 Added lines. @teolemon